go.mod

```diff
@@ -33,6 +33,7 @@ require (
 	github.com/stretchr/testify v1.6.1
 	github.com/theupdateframework/notary v0.6.1
 	golang.org/x/crypto v0.0.0-20201012173705-84dcc777aaee
+	golang.org/x/lint v0.0.0-20201208152925-83fdc39ff7b5 // indirect
```

Comment:
can you do a `go mod tidy` thanks

Reply:
ran `go mod tidy` and extra package have been removed with the latest push.